src/generic-site/content-script.js

```diff
@@ -34,5 +35,38 @@ function updateSettings(new_settings = null) {
     }
 }
 chrome.runtime.onMessage.addListener(handleMessage);
+
+function sendRollToGameLog(message) {
+    console.log(message)
+    chrome.runtime.sendMessage(message)
```

Comment:
I think we should probably have a whitelist of which actions are accepted, such as `roll`, `rendered-roll` and `hp-update`. If the `message.action` is different, it should probably block it, since it could be used to affect user settings (such as adding a malicious site to the list of accepted domains), or other internal messaging actions between beyond20 and the background script (can't remember what they are, or if they'd be an issue to let any site send those messages to the extension).